### ISSUE TITLE
Apple Pay Button is not responsive  

### ISSUE THREAD

**Issue by ramisalem** [Tap-Payments/gosellSDK-ReactNative#16]
Hello, the Apple pay button shows and works in development but after archiving it shows the button  but it is dose not show the payment sheat 

![simulator_screenshot_33F0ECA2-F360-4DCB-96BE-3C462162A370](https://user-images.githubusercontent.com/25037017/116991148-f9b74780-ac88-11eb-8537-9afb9bf2d95a.png)

https://user-images.githubusercontent.com/25037017/116991444-67fc0a00-ac89-11eb-8cc6-59becfc3d37e.MP4

**Comment by ramisalem** [comment]
@kareemTap  Do you have  any information about this issue?

**Comment by kareemTap** [comment]
@ramisalem Did you contact the support and make sure that ApplePay feature is enabled?
If yes, you need to make sure that you're using the live apple pay certificate when archiving.

**Comment by ramisalem** [comment]
@kareemTap   Yes I did enable the apple pay feature, and I have use live apple pay certificate

**Comment by kareemTap** [comment]
@ramisalem Great, so please contact our support so they will help you to fix it.

**Comment by ramisalem** [comment]
@kareemTap Thanks ,   I am trying to contact them

**Comment by AhmedHF** [comment]
> Hello, the Apple pay button shows and works in development but after archiving it shows the button but it is dose not show the payment sheat
> 
> ![simulator_screenshot_33F0ECA2-F360-4DCB-96BE-3C462162A370](https://user-images.githubusercontent.com/25037017/116991148-f9b74780-ac88-11eb-8537-9afb9bf2d95a.png)

how to integrate with apple pay?
what steps do to view the pay with Apple button?
